docs/advanced-configuration.md

```diff
@@ -399,15 +399,36 @@ does all the necessary metadata enrichment.
 
 OpenTelemetry Collector also has
 [native functionality for logs collection](https://github.com/open-telemetry/opentelemetry-log-collection).
-This chart soon will be migrated from fluentd to the OpenTelemetry logs collection.
 
-You already have an option to use OpenTelemetry logs collection instead of fluentd.
-The following configuration can be used to achieve that:
+**IMPORTANT:** This chart will be migrated from Fluentd to the OpenTelemetry logs collection by default starting from
+version 0.86.0. Please ensure that "logsEngine" parameter is explicitly set to avoid unexpected changes during upgrade.
+
+The following configuration can be used to switch between fluentd and OpenTelemetry logs collection:
 
```

Comment:
Suggestion:

**IMPORTANT:** This chart will be migrated from Fluentd to the OpenTelemetry logs collection, which will be the default starting from version 0.86.0. Please ensure that the ``logsEngine`` parameter is explicitly set to avoid unexpected changes during upgrade.

Use the following configuration to switch between Fluentd and OpenTelemetry logs collection: